CHANGELOG.md

```diff
@@ -4,12 +4,16 @@ All notable changes to Hylo Evo (the Hylo front-end) will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
ooo, should I be adding to this when I do new features?

Reply:
yes ideally!